releasetasks/templates/fennec/source.yml.tmpl

```diff
@@ -0,0 +1,256 @@
+            - queue:create-task:aws-provisioner-v1/opt-linux64
+            - queue:define-task:aws-provisioner-v1/build-c4-2xlarge
+            - queue:create-task:aws-provisioner-v1/build-c4-2xlarge
+            - docker-worker:cache:build-{{ branch }}-release-workspace
+            - docker-worker:cache:tooltool-cache
+            - docker-worker:relengapi-proxy:tooltool.download.public
+
+        payload:
+            maxRunTime: 7200
+            image: rail/source-builder@sha256:20ea2d9aff06740d6a35ff2f42af98c8d8b65efec9d0dea48692fbb3025d5d8c
+            cache:
+                build-{{ branch }}-release-workspace: /home/worker/workspace
+                tooltool-cache: /home/worker/tooltool-cache
+                tc-vcs: /home/worker/.tc-vcs
+            artifacts:
+                public/build:
+                    path: /home/worker/artifacts/
+                    expires: {{ now.replace(days=365) }}
+                    type: directory
+            command:
+                - /bin/bash
+                - -c
+                - bin/build.sh || JOB_SCRIPT=testing/taskcluster/scripts/builder/build-linux.sh bin/build.sh
+            env:
+                GECKO_REPOSITORY: "https://hg.mozilla.org/{{ repo_path }}"
+                GECKO_HEAD_REV: "{{ revision }}"
+                MOZHARNESS_SCRIPT: "mozharness/scripts/fx_desktop_build.py"
+                MOZHARNESS_CONFIG: "builds/releng_sub_linux_configs/64_source.py"
+                # MH_BRANCH is used for branch specific MH configs
+                MH_BRANCH: "{{ branch }}"
+                MH_BUILD_POOL: "taskcluster"
+                # Don't clone mozharness, use the in-tree version
+                MOZHARNESS_DISABLE: "true"
+                MOZ_PKG_VERSION: "{{ version }}"
+                MOZHARNESS_ACTIONS: "no-clone-tools no-checkout-sources no-setup-mock"
+        metadata:
+            owner: release@mozilla.com
+            source: https://github.com/mozilla/releasetasks
+            name: "{{ branch }} source generation"
+            description: |
+                Generates a source tarball
+
+        routes:
+            - tc-treeherder-stage.v2.{{ branch }}.{{ revision }}.{{ pushlog_id }}
+            - tc-treeherder.v2.{{ branch }}.{{ revision }}.{{ pushlog_id }}
+            - index.releases.v1.{{ branch }}.{{ revision }}.{{ product }}.{{ version | replace(".", "_") }}.build{{ buildNumber }}.source_tarball
+            - index.releases.v1.{{ branch }}.latest.{{ product }}.latest.source_tarball
+        extra:
+            {{ common_extras(taskname=buildername, locales=["null"], platform="null") | indent(12)}}
+            treeherderEnv:
+                - staging
+                - production
+            treeherder:
+                symbol: Src
+                groupSymbol: Release
+                collection:
+                    opt: true
+                machine:
+                    platform: linux64
+                build:
+                    platform: linux64
```

Comment:
Can you also adjust the treeherder information, so we don't post the results to linux64.